test/python/WMCore_t/WorkQueue_t/WorkQueue_t.py

```diff
@@ -602,44 +611,43 @@ def testQueueChaining(self):
                                        {})
         self.assertEqual(0, len(self.localQueue))
         self.assertEqual(2, len(work))
-
+ 
```

Comment:
Actually it is not  that, it seems it added only one space for each line. Not sure how that happens. Anyway, fixed and updated.